cfme/tests/pod/test_appliance_crud.py

```diff
@@ -84,20 +85,37 @@ def appliance_data(provider):
     return app_data
 
 
+def read_host_file(appliance, path):
+    """ Read file on the host """
+    out = appliance.ssh_client.run_command('cat {}'.format(path), ensure_host=True)
+    if out.failed:
+        pytest.fail("Can't read pvc file")
+
+    return out.output
+
+
 @pytest.fixture
-def template_tags(appliance):
+def template(appliance):
     try:
         api = trackerbot.api()
         stream = get_stream(appliance.version.vstring)
         template_data = trackerbot.latest_template(api, stream)
-        template = api.template(template_data['latest_template']).get()
-        # TODO: fix this in trackerbot by adding appropriate serialization to Template
-        tags = json.loads(template['custom_data'].replace("'", '"').replace('u"', '"'))
-        return tags['TAGS']
+        return api.template(template_data['latest_template']).get()
     except BaseException:
         pytest.skip("trackerbot is unreachable")
 
 
+@pytest.fixture
+def template_tags(template):
+    return template['custom_data']['TAGS']
```

Comment:
In this fixture and below, I would like to catch the potential KeyError/AttributeError exceptions that may occur when referencing trackerbot template data, or cfme_data from yaml.

Catching these exceptions will allow us to call `pytest.skip` or `pytest.fail` (I'm thinking skip is appropriate here) instead of a raw exception causing an Error result.